var/spack/repos/builtin/packages/py-radical-pilot/package.py

```diff
@@ -13,19 +13,21 @@ class PyRadicalPilot(PythonPackage):
 
     homepage = 'https://radical-cybertools.github.io'
     git      = 'https://github.com/radical-cybertools/radical.pilot.git'
-    pypi     = 'radical.pilot/radical.pilot-1.10.1.tar.gz'
+    pypi     = 'radical.pilot/radical.pilot-1.11.0.tar.gz'
 
     maintainers = ['andre-merzky']
 
     version('develop', branch='devel')
+    version('1.11.0',  sha256='71037e3438e130400ee0ee595a85bdac20ed6ef789271cd0b2298c06b5569415')
+    version('1.10.2',  sha256='56e9d8b1ce7ed05eff471d7df660e4940f485027e5f353aa36fd17425846a499')
     version('1.10.1',  sha256='003f4c519b991bded31693026b69dd51547a5a69a5f94355dc8beff766524b3c')
     version('1.9.2',   sha256='7c872ac9103a2aed0c5cd46057048a182f672191e194e0fd42794b0012e6e947')
     version('1.8.0',   sha256='a4c3bca163db61206e15a2d820d9a64e888da5c72672448ae975c26768130b9d')
     version('1.6.8',   sha256='fa8fd3f348a68b54ee8338d5c5cf1a3d99c10c0b6da804424a839239ee0d313d')
     version('1.6.7',   sha256='6ca0a3bd3cda65034fa756f37fa05681d5a43441c1605408a58364f89c627970')
 
-    depends_on('py-radical-utils@1.6.7:', type=('build', 'run'))
-    depends_on('py-radical-saga@1.6.6:',  type=('build', 'run'))
+    depends_on('py-radical-utils@1.8.4:', type=('build', 'run'))
+    depends_on('py-radical-saga@1.8.0:',  type=('build', 'run'))
```

Comment:
Does this hold also for the other versions being supported?

Reply:
@alalazo hi Massimiliano, it has backward compatibility, any RP version works with newer versions of RP and RU, but newer versions of RP are enforced to use RU>=1.8.4 and RS>=1.8.0, because of added new modules (as well as bug fixes)

should it be updated as the following?
```python
    depends_on('py-radical-utils@1.8.4:', type=('build', 'run'), when='@1.11:')
    depends_on('py-radical-saga@1.8.0:',  type=('build', 'run'), when='@1.11:')

    depends_on('py-radical-utils',        type=('build', 'run'), when='@:1.10')
    depends_on('py-radical-saga',         type=('build', 'run'), when='@:1.10')
```

Reply:
If I got correctly what you're saying, you can use the following pattern:
```python
depends_on('py-radical-utils', type=('build', 'run'))
depends_on('py-radical-saga', type=('build', 'run'))
depends_on('py-radical-utils@1.8.4:', type=('build', 'run'), when='@1.11:')
depends_on('py-radical-saga@1.8.0:',  type=('build', 'run'), when='@1.11:')
```